### PR TITLE
FIX: Advanced Registry URL connection options

### PR DESCRIPTION
**Pull request by Swaroop-Chethan**
docker-registry connection option integratio
Screenshot 2021-09-23 at 10 44 23 AM
n
Screenshot 2021-09-23 at 10 44 57 AM
Screenshot 2021-09-23 at 10 45 37 AM

Fixes: https://github.com/devtron-labs/devtron/issues/764

### CODE REVIEW

**Review by nishant-d — APPROVED** [review]

File: src/components/globalConfigurations/globalConfigurations.scss [hunk 1]
```diff
@@ -252,6 +252,8 @@
         border-radius: 8px;
         overflow: hidden;
         background: white;
+    }
+    &.collapsed-list--dash-border {
```
Comment (shivani170) [inline src/components/globalConfigurations/globalConfigurations.scss:256]: Instead can use classname=`dashed` here

Comment (Swaroop-Chethan) [reply src/components/globalConfigurations/globalConfigurations.scss:256]: okay

Comment (Swaroop-Chethan) [reply src/components/globalConfigurations/globalConfigurations.scss]: Have made changes please verify

File: src/components/dockerRegistry/Docker.tsx [hunk 2]
```diff
@@ -203,6 +236,41 @@ function DockerForm({ id, pluginId, registryUrl, registryType, awsAccessKeyId, a
...
+                    onClick={(e) => setoptionCollapsed(not)}
+                    className="rotate icon-dim-18 pointer fcn-6"
+                    style={{ ['--rotateBy' as any]: !optionCollapsed ? '-90deg' : '0deg' }}
+                />
+                <label className="fs-13 mb-0 ml-8 pointer" onClick={(e) => setoptionCollapsed(not)}>Advanced Registry URL connection options</label>
+                <a target="_blank" href="https://docs.docker.com/registry/insecure/"><Info className="icon-dim-16 ml-4 mt-5" /></a>
+            </div>}
+            {optionCollapsed &&
+                <div className="form__row ml-3" style={{ width: '100%' }}>
+                    {[{ label: 'Allow only secure connection', value: 'secure', tippy: '' }, { label: 'Allow secure connection with CA certificate', value: 'secure-with-cert', tippy: 'Use to verify self-signed TLS Certificate' }, { label: 'Allow insecure connection', value: 'insecure', tippy: 'This will enable insecure registry communication' }]
```
Comment (shivani170) [inline src/components/dockerRegistry/Docker.tsx:253]: can store this array in a variable

Comment (Swaroop-Chethan) [reply src/components/dockerRegistry/Docker.tsx]: done.

**Review by shivani170 — APPROVED** [review]

**Review by nishant-d — APPROVED** [review]

**Review by nishant-d — APPROVED** [review]

**Pull request merged**